contracts/interfaces/LockableSeed.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.23;
+
+import './Lockable.sol';
+
+
+contract LockableSeed is Lockable {
+  uint private seed;
```

Comment:
you could just make this `public` as the compiler will automatically create a getter function for every public state variable available but not a setter.  you can thus still restrict the `setSeed` to `onlyOwner` without issue.

Reply:
opps, I misssed that ty.